### DISCUSSION TITLE
How to change <C-y> for accepting suggestions

### DISCUSSION THREAD

**Discussion by ak-seyam** [neoclide/coc.nvim#3439] [Q&A] [answered]
I don't know what command should do this or even what this action is called but C-y seems to be mapped to something else by default in neovim so it takes some time, a second or so, to accept that suggestion I want to change it to <leader-y> for example. What should I add to my init file?

Thanks

**Comment by chemzqm** [comment, marked as answer]
Could be remapped by some vim plugin try`:verbose imap <C-y>`
